Engine/Game/GameScene.cpp

```
@@ -0,0 +1,13 @@
+#include "GameScene.h"
+
+void GameScene::Initialize() {
```

cppcheck (2.10-2)

[style] The function 'Initialize' is never used.

(unusedFunction)